src/styles/components/_modal.scss

```diff
@@ -29,6 +29,11 @@ $component-name: modal;
       flex: 1 auto;
       overflow-x: hidden;
       overflow-y: scroll;
+
+      .button__container {
```

Comment:
I don't think this belongs here. We try to use hyphenated bem styles, with a file per component, i.e.:
```
.component__sub-component--variants {
   ...
}
`````
This would mean in your case, I think...
`className="form__button-container"`

// _form.scss
```
.form {
  &__button-container {
     ...
  }
}
```